chains/evm/calls/calls.go

```diff
@@ -22,7 +23,7 @@ type ContractCaller interface {
 }
 
 type GasPricer interface {
-	GasPrice() ([]*big.Int, error)
+	GasPrice(priority uint8) ([]*big.Int, error)
```

Comment:
Would probably use something like this https://www.sohamkamani.com/golang/options-pattern/ to avoid having magic number in functions and allow for future options